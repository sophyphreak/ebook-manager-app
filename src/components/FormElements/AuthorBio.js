import React, { Component } from 'react';

const AuthorBio = (props) => (
  <label>
    Author Biography:
    <input type="text" value={props.value} onChange={props.onChange} />
  </label>
);

export default AuthorBio;