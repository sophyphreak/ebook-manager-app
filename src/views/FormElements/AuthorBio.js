import React, { Component } from 'react';

const AuthorBio = (props) => (
  <label>
    Author Biography:
    <textarea type="text" value={props.value} onChange={props.onChange} />
  </label>
);

export default AuthorBio;