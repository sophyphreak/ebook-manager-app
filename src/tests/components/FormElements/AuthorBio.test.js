import React from 'react';
import { shallow } from 'enzyme';
import AuthorBio from '../../../components/FormElements/AuthorBio';

let wrapper;

beforeEach(() => {
  wrapper = shallow(<AuthorBio />);
});

test('should render AuthorBio correctly', () => {
  expect(wrapper).toMatchSnapshot();
});
