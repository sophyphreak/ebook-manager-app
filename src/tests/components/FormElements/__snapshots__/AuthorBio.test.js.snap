// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render AuthorBio correctly 1`] = `
<TextAreaInput
  label="Author Biography:*"
/>
`;
